template/runner/graph_classification_iamdb/graph_classification_iamdb_inference.py

```diff
@@ -0,0 +1,10 @@
+# DeepDIVA
+from template.runner.base import BaseInference
+from template.runner.image_classification.setup import ImageClassificationSetup
+
+
+class GraphClassificationInference(BaseInference):
+
+    def __init__(self):
+        super().__init__()
+        self.setup = ImageClassificationSetup()
```

Comment:
this should not be done, as if something is changed in ImageClassification it breaks here and this violates the runner incapsulation design

Reply:
I have not actually tested/implemented this. I will delete this file for now and implement and test later.

Reply:
Good solution. Then you can delete the file and commit(+push) again